test/e2e/lib/jest/globalSetup.ts

```diff
@@ -0,0 +1,65 @@
+import { mkdir } from 'fs/promises';
+import path from 'path';
+import config from 'config';
+import { chromium } from 'playwright';
+
+// Save cookies for defaultUser.
+export default async (): Promise< void > => {
+	if ( process.env.SAVE_AUTH_COOKIES !== 'true' ) {
+		return;
+	}
+
+	// If configuration file is not found, log a message and exit this setup.
+	if ( config.util.equalsDeep( config.util.toObject(), {} ) ) {
+		console.error( 'No decrypted configuration file was found.' );
+		return;
+	}
+
+	const browser = await chromium.launch();
+	const calypsoBaseURL = config.get( 'calypsoBaseURL' );
+
+	// Create a directory to store the generated cookie file.
+	// The cookie file will be stored in the `test/e2e/cookies` directory.
+	const cookieBasePath = path.join( __dirname, '../../', 'cookies' );
+	await mkdir( cookieBasePath, { recursive: true } );
+	// This is important!
+	// Jest does not permit values created in globalSetup/globalTeardown to be read
+	// by the test suites themselves. See https://jestjs.io/docs/configuration#globalsetup-string.
+	// As such the only way to communicate the cookie path to the test suites is to
+	// set this environment variable and have the `@automattic/calypso-e2e` module check for
+	// its presence when running each test file.
+	process.env.COOKIES_PATH = cookieBasePath;
+
+	const userList = [ 'gutenbergSimpleSiteUser', 'wooCommerceUser', 'defaultUser' ];
```

Comment:
Do we want the new `eCommerceUser` here instead of the woo one?

Reply:
I'll rebase and switch over the user. Thanks for catching!

Reply:
Ah shoot, forgot to do this in the shuffle of things. Will address in next patch.